app/assets/javascripts/student_profile/page_container.js

```diff
@@ -114,8 +114,8 @@
       });
     },
 
-    onSaveServiceFail: function(request, status, message) {
-      this.setState({ requests: merge(this.state.requests, { saveService: 'error' }) });
+    onSaveServiceFail: function(response, status, message) {
+      this.setState({ requests: merge(this.state.requests, { saveService: response.responseJSON.errors[0]}) });
```

Comment:
@awalker4 One thought here, which is not clear at all from the code, is that these objects right now all hold string keys representing the state of the request.  That's what the UI code is branching on later on, as you awesomely discovered and updated.

I think your approach makes sense here, but any other thoughts you have on how to clarify this in the future are awesome too :)